samtranslator/open_api/open_api.py

```diff
@@ -450,6 +450,17 @@ def add_tags(self, tags):
         :param dict tags: dictionary of tagName:tagValue pairs.
         """
         for name, value in tags.items():
+            # verify the tags definition is in the right format
+            if not isinstance(self.tags, list):
+                raise InvalidDocumentException(
+                    [
+                        InvalidTemplateException(
+                            "Tags in open api definition needs to be a list. {} is a {} not a list.".format(
+                                self.tags, type(self.tags)
```

Comment:
Perhaps better to get rid of the weird Pythonism (`<class 'dict'>`)?

```suggestion
                                self.tags, type(self.tags).__name__
```

Reply:
Sure yes that makes the message more readable! Will add in this change